database/migrations/2023_07_07_084116_create_employees_table.php

```diff
@@ -10,14 +10,16 @@
      * Run the migrations.
      */
     public function up(): void
-    {   
-       
+    {
+
         Schema::create('employees', function (Blueprint $table) {
             $table->id();
             $table->string('name');
             $table->timestamps();
             $table->boolean('praesentiert')->default(false);
             $table->boolean('gekocht')->default(false);
+            $table->boolean('is_available')->default(true);
```

Comment:
please create a new migration that modifies the column and revert this change
https://laravel.com/docs/11.x/migrations#modifying-columns

On deployment only new Migrations are executed